packages/appium/lib/appium.js

```diff
@@ -241,6 +251,261 @@ class AppiumDriver extends DriverCore {
     }
   }
 
+  /**
+   * Initialize a new bidi connection
+   * @param {import('ws').WebSocket} ws The websocket connection object
+   * @param {import('http').IncomingMessage} req The connection pathname, which might include the session id
+   */
+  async onBidiConnection(ws, req) {
+    ws.on('error', (err) => {
+      // Can't do much with random errors on the connection other than log them
+      this.log.error(err);
```

Comment:
It makes sense to log session identifier if possible, so we can always connect particular errors with corresponding sessions.

Reply:
same below